### PR TITLE
Urgent known issue for OCP 4.7

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1929463

Preview: https://deploy-preview-29686--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-7-release-notes.html#ocp-4-7-known-issues

### CODE REVIEW

**Comment by codyhoag** [comment]
@mffiedler PTAL. Thanks!

**Review by mffiedler — APPROVED** [review]
LGTM

File: release_notes/ocp-4-7-release-notes.adoc [hunk 1]
```diff
@@ -1945,6 +1945,15 @@ This script removes unauthenticated subjects from the following cluster role bin
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1821771[*BZ#1821771*])
 
+* Starting and stopping pods rapidly can result in pods getting stuck in the `Terminating` state. As a workaround, you must remove the terminating pod by running the following command:
```
Comment (kalexand-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1948]: Maybe
```suggestion
* Starting and stopping pods rapidly can result in pods getting stuck in the `Terminating` state. As a workaround, you must remove the stuck pod by running the following command:
```

File: release_notes/ocp-4-7-release-notes.adoc [hunk 2]
```diff
@@ -1945,6 +1945,15 @@ This script removes unauthenticated subjects from the following cluster role bin
...
++
+[source,terminal]
+----
+$ oc delete --force -n <project_name> <pod_name>
+----
++
+This will be fixed in a future release of {product-title}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929463[*BZ#1929463*])
```
Comment (kalexand-rh) [inline release_notes/ocp-4-7-release-notes.adoc:1955]:
```suggestion
This issue will be fixed in a future release of {product-title}. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1929463[*BZ#1929463*])
```

**Review by kalexand-rh — COMMENTED** [review]
I have two suggestions, but this LGTM!

**Review by vikram-redhat — APPROVED** [review]
lgtm for merge

**Pull request merged**